@xen-orchestra/proxy/src/app/_debounceResource.js

```diff
@@ -12,24 +14,27 @@ export async function debounceResource(pDisposable, hooks, delay = 0) {
     if (timeoutId !== undefined) {
       clearTimeout(timeoutId)
       timeoutId = undefined
-      hooks.removeListener('stop', onStop)
 
       return disposable.dispose()
     }
   }
 
-  const onStop = () => {
+  disposers.add(() => {
```

Comment:
Disposer is not removed when the resource is already disposed!

Reply:
This case is properly handled, the real dispose will be called once. But, we can remove the dispose in this case.